apiserver/service/rag_server.go

```diff
@@ -123,6 +123,36 @@ func (r *RagAPI) ReportDetail(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, result)
 }
 
+// @Summary	Get scatter data of a rag
+// @Schemes
+// @Description	Get scatter data of a rag
+// @Tags			RAG
+// @Accept			json
+// @Produce		json
+// @Param			ragName		query		string	true	"rag name"
+// @Param			namespace	header		string	true	"Name of the bucket"
+// @Param			appName		query		string	true	"application name"
```

Comment:
@0xff-dev  should we remove this appName?

Reply:
No.  `appName` is part of the path to the minio file.